lib/loader.js

```diff
@@ -12,7 +11,21 @@ const pify = require("pify");
 // fs tasks when running the custom importer code.
 // This can be removed as soon as node-sass implements a fix for this.
 const threadPoolSize = process.env.UV_THREADPOOL_SIZE || 4;
-const asyncSassJobQueue = async.queue(sass.render, threadPoolSize - 1);
+let asyncSassJobQueue = null;
+let loadingError = null;
+
+try {
+    const sass = require("node-sass");
+    const sassVersion = require("node-sass/package.json").version;
+
+    if (/^4[.]/.test(sassVersion)) {
```

Comment:
I think better to check `>= 4`, maybe in future we can have `5`

Reply:
Yep, please check for greater than  

```js
if (Number(sassVersion[0]) >= 4) {}
```

Reply:
What if we're not compatible with the `node-sass` v5 API?